### PR TITLE
Update requests-kerberos to version 0.13.0

### PR DESCRIPTION
**Pull request by ejegrova**
Newer version of requests-kerberos is compatible with
changes in Python 3.10.

Signed-off-by: ejegrova <ejegrova@redhat.com>

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] OpenAPI schema is updated (if applicable)
- [ ] DB schema change has corresponding DB migration (if applicable)
- [ ] README updated (if worker configuration changed, or if applicable)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.8
```
Comment (brunoapimentel) [inline requirements.txt:2]: I think this does not change anything, but we changed the pip-compile python version here.

**Comment by brunoapimentel** [comment]
Wdyt about dropping a note in the commit message to say that this upgrade is necessary because of Python 3.10 compatibility?

File: requirements.txt (hunk 1 above)
Comment (ejegrova) [reply requirements.txt:2]: yes, it's probably because my default python is 3.8. I'll generate this with python 3.9

**Comment by ejegrova** [comment]
> Wdyt about dropping a note in the commit message to say that this upgrade is necessary because of Python 3.10 compatibility?

I thought it says that. Can you be more exact? Do you want me to also write there that previous versions are not compatible?

File: requirements.txt (hunk 1 above)
Comment (ejegrova) [reply requirements.txt]: done

**Review by brunoapimentel — APPROVED** [review]
LGTM :rocket:

**Review by akhmanova — APPROVED** [review]

**Pull request merged**